packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -390,7 +391,9 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
   const esqlCallbacks: ESQLCallbacks = useMemo(
     () => ({
       getSources: async () => {
-        return await getIndicesList(dataViews);
+        const remoteIndices = await getRemoteIndicesList(dataViews);
+        const normalIndices = await getIndicesList(dataViews);
```

Comment:
Any reason not to execute these in parallel?

e.g.  `const [remoteIndices, localIndices] = Promise.all(getRem....)`

Not sure on the performance gains, but might be something 🤷

Reply:
Sure, y not, done https://github.com/elastic/kibana/pull/180610/commits/a62e69a398e0f0cea8f3a8036c305844c5e5dfdc